app/models/contact.rb

```diff
@@ -1,6 +1,10 @@
 class Contact < ActiveRecord::Base
+  EMBEDDED_CONTACT_REGEXP = /\[Contact\:([0-9]+)\]/
```

Comment:
That's because we have to reference it from `lib/govspeak/contacts_extractor.rb`, which won't be possible without `include GovspeakHelper` in the contacts extractor. It seems odd to include a view helper in `/lib`.